src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ResourceUtilizationInstruments.cs

```diff
@@ -13,18 +15,42 @@ namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
 internal static class ResourceUtilizationInstruments
```

Comment:
Any specific reason for this type to be non-public?
This, essentially, makes all of these const unavailable to consumers, who will have to hard-code those in their code. E.g., in a sample I had to do the following:
![image](https://github.com/user-attachments/assets/95c1ef06-563d-4495-99c5-da4f31370e63)

Reply:
Good point, but I don't know the history, it might be a common reason "let's not expose it until someone requests it"

I can make it `public` in a separate PR and mark it Experimental, unless you have a different point of view?

Reply:
I don't think marking it as experimental would be helpful or useful as it would affect the UX and devex.
Let's sync on this.